### DISCUSSION TITLE
How to load restricted views

### DISCUSSION THREAD

**Discussion by mystycs** [jqnatividad/ckanext-officedocs#21] [Q&A]
Has anyone figured out a good method to allowing restricted views to load in the officeviewer extension?

**Comment by jqnatividad** [comment]
One avenue that may be worthwhile to explore is Chrome's **built-in** Office Viewer -

https://chrome.google.com/webstore/detail/office-editing-for-docs-s/gbkeegbaiigmenfmjfclcdgdpimamgkj?hl=en

If your user population is using Chrome, then this should work without having to worry about making the resource public.
